### DISCUSSION TITLE
any plan to refactor with Floating UI?

### DISCUSSION THREAD

**Discussion by chshouyu** [atomiks/tippyjs#1035] [Ideas]
popperjs has refactored to https://github.com/floating-ui/floating-ui

**Comment by atomiks** [comment]
I don't really have plans to continue working on Tippy anymore except to fix bugs (which I also don't have much time for 😅 ). 

Floating UI is going to export the work of Tippy soon (although, only React for now). It will be much better architecturally and support many more type of components in an accessible manner that Tippy fails to do. Since it only supports React and will be 100% unstyled it won't really replace Tippy completely though.
